src/bot.py

```diff
@@ -141,10 +145,15 @@ def int_conv(string: str) -> bool:
 
     # log comment
 
-    if (bonusNugs == 0):
+    if bonus_nugs:
         comment.reply("SUCCESS_MESSAGE")
     else:
         comment.reply("SUCCESS_MESSAGE")
 
-    for post in reddit.subreddit("xeothtest").new(limit=10):
-        comment.mod.distinguish(os.getenv('STICKIED_MESSAGE'), sticky=True)
+for post in reddit.subreddit("xeothtest").new():
+    if db.check_post(post.id): # breaks completely when posts previously processed are reached
+        break
+
+    db.add_post(post.id)
+
+    comment.mod.distinguish(os.getenv('STICKIED_MESSAGE'), sticky=True)
```

Comment:
I was thinking: maybe we should use AutoModerator to pin a comment and then only check whether the comments were made under it? This would save quota and wouldn't rate limit us.

Reply:
Rate limiting's not a big deal, we only need a few upvotes. AutoMod can do the sticky if we really can't get it working, and isn't much effort to change anyway.